src/it/scala/net/snowflake/spark/snowflake/InjectTestSuite01.scala

```diff
@@ -73,12 +73,29 @@ class InjectTestSuite01 extends IntegrationSuiteBase {
   }
 
   test("inject exceptions for Arrow read") {
+    // Configure some spark options
+    // manually check these options are sent correctly.
```

Comment:
Manually check ?

Reply:
@sfc-gh-sshankar The message is sent to telemetry in snowhouse. Currently, there is no way to login to snowhouse in IT test. So, we can only do manual test.

Reply:
@sfc-gh-sshankar After consideration, I think we can use Unit test instead of integration test to test spark options are included in the telemetry message. I have updated the PR. Could you please take a look?